style.css

```diff
@@ -0,0 +1,84 @@
+@import url("https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@0,400;0,500;0,600;0,700;0,800;0,900;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  box-sizing: border-box;
+  background-image: url("material1.png");
+  background-repeat: no-repeat;
+  background-size: 35%;
+  background-position: left top;
+}
+
+header {
+  display: flex;
+  margin: 2%;
+  justify-content: end;
+}
+
+.fa-bars {
+  position: inherit;
+  display: flex;
+  font-size: 2em;
+  justify-content: end;
+  margin: 15% 15px;
+  padding-top: 3%;
+}
+
+.logo {
+  font-family: "Playfair Display", serif;
+  font-size: 4em;
+  font-weight: normal;
+  letter-spacing: 2px;
```

Comment:
Nice job with the headline section. I would like to suggest that you satisfy one of the project requirements that says you should stick to the Figma template design as much as possible. This includes using the same fonts, colors, and margins.
- [ ] Kindly use the `color: 1a2236` and `font-size: 50px` or `3.125rem` for this CSS declaration.

Reply:
Chenged